kubeflow/components/filter-paraphrase.yaml

```diff
@@ -38,13 +38,13 @@ implementation:
       mkdir -p output_dataset
       cp -r input_dataset/* output_dataset
       
-      s3_output_dir="$s3_input_datadir" # filtering does not create a new folder
+      s3_output_dir="s3://${s3_bucket}/${owner}/dataset/${project}/${experiment}/${dataset}-gen-filt/`date +%s`/"
             
       mkdir -p `dirname $s3_output_datadir`
       echo ${s3_output_dir} > $s3_output_datadir
 
       paraphrasing_arguments="$@"
-      if [[ "$task_name" == "almond_dialogue_nlu"* ]] ; then # almond_dialogue_nlu or almond_dialogue_nlu_agent
+      if "$task_name" | grep -q "almond_dialogue_nlu" ; then # almond_dialogue_nlu or almond_dialogue_nlu_agent
```

Comment:
oh, I didn't realize the goal was to match both almond_dialogue_nlu and almond_dialogue_nlu_agent
I would use a `case` statement in that case. Otherwise, you need an `echo` in front of `$task_name`